source/js/main.js

```diff
@@ -146,6 +146,56 @@ let main = {
       MozfestCarousels.init();
       MozfestHeroCarousels.init();
     }
+
+    if (document.querySelector(`.donate-banner`)) {
+      document
+        .querySelector(`.donate-banner .banner-close`)
+        .addEventListener(`click`, (e) => {
```

Comment:
Let's take advantage of `?` here:

```suggestion
    document
        .querySelector(`.donate-banner .banner-close`)?
        .addEventListener(`click`, (e) => {
            // skip the banner if it got dismissed by the user today already
            if (parseInt(localStorage.getItem(DISMISS_KEY)) === getDay()) return;
            localStorage.removeItem(DISMISS_KEY);
            
            const banner = e.target.parentNode;
```

And then we can use `banner` directly, rather than needing to requery the document for `.donate-banner` in the various places.